src/main/java/no/nav/veilarbperson/config/ApplicationConfig.java

```diff
@@ -41,15 +41,6 @@ public class ApplicationConfig {
 			.expireAfterWrite(Duration.ofMinutes(30))
 			.build();
 
-    @Bean
```

Comment:
Må nok vente litt med å merge dette, da dette faktisk blir brukt for et endepunkt